test/e2e/components/vl-element.js

```diff
@@ -14,6 +14,9 @@ class VlElement extends WebElement {
       if (await this.driver.executeScript('return arguments[0].shadowRoot != undefined', this)) {
         this.shadowRoot = await new VlElement(this.driver, await this.driver.executeScript('return arguments[0].shadowRoot.lastElementChild', this));
       }
+      if (mixins && Array.isArray(mixins)) {
+        mixins.forEach((mixin) => Object.assign(this, mixin));
+      }
```

Comment:
Hier is dus die constructor waarnaar ik verwees in pattern review. Merk nu wel dat het voor testen is. Maar de vraag is dus wat als mixins zelfde functie hebben, wordt die dan overschreven?

Reply:
Bij de e2e testen worden de mixins via constructie meegegeven, de vlFormValidation en vlPattern zijn voorlopig de enige mixins die toegepast worden en bevatten beide unieke functies. Een prefix zal geïntroduceerd worden indien nodig.